lib/ask/runtime/verboice_channel.ex

```diff
@@ -383,8 +383,9 @@ defmodule Ask.Runtime.VerboiceChannel do
         {:ok, %{"state" => "completed"}} -> false
         {:ok, %{"state" => "failed"}} -> false
         {:ok, %{"state" => "canceled"}} -> false
-        {:ok, %{"state" => "expired"}} -> false
-        {:ok, %{"state" => _}} -> true
+        {:ok, %{"state" => _}} ->
+          SurvedaMetrics.increment_counter_with_label(:surveda_verboice_enqueue, [elem(response, 0)])
```

Comment:
This is not where a call is enqueued in Verboice. Move this to the `setup` function.